WooCommerce/src/main/kotlin/com/woocommerce/android/ui/mystore/MyStoreViewModel.kt

```diff
@@ -202,7 +202,11 @@ class MyStoreViewModel @Inject constructor(
             selectedSite.observe()
                 .filter { it?.isJetpackConnected == true }
                 .take(1)
-                .collect { loadStoreStats(activeStatsGranularity.value) }
+                .collect {
+                    activeStatsGranularity.value?.let {
+                        loadStoreStats(it)
+                    }
```

Comment:
@JorgeMucientes @hichamboushaba is that okay? I believe this value started to be nullable in 09609852557b476d6d43aa9699f872f25bd4a5be commit

Reply:
Thanks for the heads up @wzieba. An alternative would be to use `_activeStatsGranularity.value` which is not nullable. Wdyt @hichamboushaba ?

```suggestion
                    loadStoreStats(_activeStatsGranularity.value)
```